### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by ex-ml** [mattlongman/Hassio-Access-Point#57] [General]
I created this repo as a fork of a previously abandoned project, but I find that I no longer have the time to actively respond to issues and PRs. Additionally, as I no longer use this myself, I don't have the facilities to properly test changes. I did start working towards creating test/dev branch releases to test changes, but without anyone else testing (different use-cases, hardware, architecture), it still comes back to me pushing an update and hoping it doesn't break things for anyone 😨 ... As such, I've backed off of making any big changes.

I've also considered changing to read-only/EoL to encourage someone more capable to fork the repo and continue development, but as there is still a surprising amount of usage, this would inconvenience people and force a migration.

Why am I explaining this?

Ideally, I'm looking for active users who can assist with issues, submitting PRs and testing. I'd be happy to run through the components of the addon and explain how it works (to the best of my ability -- this addon is just some cobbled together software, so I'm by no means an expert).

**Comment by rrooggiieerr** [comment]
@ex-ml I can relate to your struggles maintaining a project. I've got some projects myself and it's not always easy to find the time and lack of testing equipment is sometimes also an issue. As an hobbyist you can't have a bunch of hardware laying around to test all kind of use cases which you're not facing yourself. I'm happy to be one of the testers of new releases, especially since I have a HA test environment for my own developments on a Pi 3 which is now also running your Hassio-Access-Point Add-on

**Comment by ROBOT0-VT** [comment]
Hiya, I've been working on this repo to fix some issues I've encountered with it. I've already made decent progress into reworking it to be easier to maintain, so I'd be happy to lend a hand. I've already made one PR, and more will arrive over the next few days :D

**Reply by ex-ml** [reply]
Thanks for the PR and note here @ROBOT0-VT. I'll check it out as soon as I can, and will add you as a maintainer. 

One question, any particular reason for bumping the version to 1.0?

**Reply by ROBOT0-VT** [reply]
The validation for booleans is a breaking change, requiring users to reset their configs. I figured that would qualify as a "major" change by semantic versioning.

I guess I could add validation code to account for it, just wasn't my focus while working on it

**Reply by rrooggiieerr** [reply]
To not making it a breaking change you could detect if there is a `config.json` and if so migrate to the new format

**Reply by ROBOT0-VT** [reply]
Commented on the change in the PR :D

**Reply by ex-ml** [reply]
I've updated your PR. Happy for you to merge when it's ready as you and @rrooggiieerr have done all of the work. In terms of maintaining, I've just added a Discord server: https://discord.gg/jhvxavYq - maybe we can use this for active maintainers, but for now, would be good to get both of you on there to work out what to do with this repo.
